src/NzbDrone.Core/MediaFiles/EpisodeFileMovingService.cs

```diff
@@ -51,12 +53,13 @@ public class EpisodeFileMovingService : IMoveEpisodeFiles
             _diskTransferService = diskTransferService;
             _diskProvider = diskProvider;
             _mediaFileAttributeService = mediaFileAttributeService;
+            _scriptImportDecider = scriptImportDecider;
             _eventAggregator = eventAggregator;
             _configService = configService;
             _logger = logger;
         }
 
-        public EpisodeFile MoveEpisodeFile(EpisodeFile episodeFile, Series series)
+        public (EpisodeFile, bool) MoveEpisodeFile(EpisodeFile episodeFile, Series series)
```

Comment:
This case is only used for existing episode files, already in the series folder, I don't think we want to handle these renames with an "import" script.

Reply:
I noticed the same thing. I've been wondering if really the `EpisodeFileMover` should be broken up into two classes, the `EpisodeFikeMover` itself and perhaps an `EpisodeFileRenamer`, probably still sharing an abstract base with `TransferFile`. At one point I was running into recursive dependencies die zu that but that rectified itself in the process of fixing other issues. I might still break it up entirely if it becomes necessary.